app/robot-shop/helm/values.yaml

```diff
@@ -127,6 +127,13 @@ payment:
 
 rabbitmq:
   replicas: 3
+  resources:
+    requests:
+      cpu: 500m
+      memory: 2Gi
+    limits:
+      cpu: 1
+      memory: 2Gi
   tolerations:
```

Comment:
Why are we doing workload specific tolerations? RabbitMQ should tolerare the `key: persistent, value: true` taint on the `persistent-ng` nodegroup.

Reply:
will create a separate PR for this, where will rename it to persistence and move all app persistence layer to it